roles/app/defaults/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+s3_headers: "{\"Cache-Control":"max-age=31557600\"}"
```

Comment:
The build is failing due to this line. I think we either need to escape all of the double quotes, or wrap in single quotes:
```yaml
"{\"Cache-Control\":\"max-age=31557600\"}"
```
```yaml
'{"Cache-Control":"max-age=31557600"}'
```

Reply:
:+1: thanks, fixed.